sls/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.9
+
+# Set envs
+ENV CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true \
+  PATH=$PATH:/usr/local/gcloud/google-cloud-sdk/bin
+
+# Install dependencies and cleanup
+RUN apk add --update curl nodejs=10.14.2-r0 nodejs-npm=10.14.2-r0 python3=3.6.8-r1 \
+  && rm -rf /var/cache/apk/* \
+  && curl -Ss https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-233.0.0-linux-x86_64.tar.gz > /tmp/google-cloud-sdk.tar.gz \
+  && mkdir -p /usr/local/gcloud \
+  && tar -C /usr/local/gcloud -xf /tmp/google-cloud-sdk.tar.gz \
+  && /usr/local/gcloud/google-cloud-sdk/install.sh \
+  && rm /tmp/google-cloud-sdk.tar.gz
+
+# Install gcloud beta and serverless CLI
+RUN gcloud components update \
+  && gcloud components install beta \
+  && npm config set unsafe-perm true \
+  && npm install -g serverless@1.37.1
```

Comment:
1. please extract 1.37.1 to an env var at the top to make version bumps easier
2. please add [our standard `LABEL`s](https://github.com/kiwicom/dockerfiles/blob/64bb655fa4d70309da0820c59a0cd7c884fd0e9a/sonar-scanner/Dockerfile#L19), such as
```
LABEL name=sonar-scanner version=$VERSION \
      maintainer="Bence Nagy <bence@kiwi.com>"
```